apps/ideaspace/src/components/modules/SaveIdea/SaveIdea.js

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import SaveButton, { SaveButtonProps } from '@devlaunchers/components/src/components/molecules/InteractionButtons/SaveButton';
+import { StyledSaveIdea } from './StyledSaveIdea';
+
+
+const SaveIdea = ({savedCards, setSavedCards, id}) =>{
+    
+    return(
+        <StyledSaveIdea>
+            <SaveButton
+            disabled = {savedCards.includes(id)}
```

Comment:
it might help to extract this `savedCards.includes(id)` to a variable on line 7, since it is repeated on the next 2 lines.

Reply:
Thanks, Dave. I will update the changes.

Reply:
The updates have been implemented.